Convex_hull_3/doc/Convex_hull_3/Doxyfile.in

```diff
@@ -1,3 +1,9 @@
 @INCLUDE = ${CGAL_DOC_PACKAGE_DEFAULTS}
 
 PROJECT_NAME = "CGAL ${CGAL_DOC_VERSION} - 3D Convex Hulls"
+
+\# macros to be used inside the code
+ALIASES += "cgalNamedParamsBegin=<dl class=\"params\"><dt>Named Parameters</dt><dd> <table class=\"params\">"
+ALIASES += "cgalNamedParamsEnd=</table> </dd> </dl>"
+ALIASES += "cgalParamBegin{1}=<tr><td class=\"paramname\">\ref BGL_\1 \"\1\"</td><td>"
+ALIASES += "cgalParamEnd=</td></tr>"
```

Comment:
The problem was that you didn't call `cgalNamedParamsEnd`, and not this. Should have checked the compiled doc, again.

Reply:
Actually, yes and no. If you did check the compiled doc, you would have seen that now, it actually calls the NP instead of the `{blahblah}` version, which is what I was looking for. Now, the missing `cgalNamedParamsEnd` explains the weird indentation, which I thought was because of the \attention that I'm not used to use.